scripts/pages/settings/mouse-sensitivity.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+
+const CM_CONVERSION_FACTOR = 2.54;
+
+class MouseSensitivity {
+	static panels = {
+		/** @type {TextEntry} @static */
+		dpi: $('#DPI'),
+		/** @type {DropDown} @static */
+		unitSelection: $('#UnitSelection'),
+		/** @type {TextEntry} @static */
+		unitsPer360: $('#UnitsPer360')
+	}
+	
+	static calculateSensitivity(){
+		if(!this.panels.dpi.text) return;
+		
+		const yaw = GameInterfaceAPI.GetSettingFloat('m_yaw')
+		const eDPI = this.panels.dpi.text * GameInterfaceAPI.GetSettingFloat('sensitivity');
+		let per360 = (360 / (eDPI * yaw));
+
+		if(this.panels.unitSelection.GetSelected().id === 'm_sens_cm')
+		per360 *= CM_CONVERSION_FACTOR;
+		
+		//displayprecision attribute does not work on TextEntry
+		per360 = Math.round(per360 * 100)/100;
+		this.panels.unitsPer360.text = per360;
+	}
+	
+	static setSensitivity(){
+		if(!this.panels.dpi.text) return;
+		
+		const yaw = GameInterfaceAPI.GetSettingFloat('m_yaw')
+		const eDPI = (360 / yaw) / this.panels.unitsPer360.text;
+		let sensitivity = eDPI / this.panels.dpi.text;
+		
+		if(this.panels.unitSelection.GetSelected().id === 'm_sens_cm')
+		sensitivity *= CM_CONVERSION_FACTOR;
+		
+		GameInterfaceAPI.SetSettingFloat('sensitivity', sensitivity);
+	}
+	
+	static saveDPI(){
+		$.persistentStorage.setItem('settings.dpi', this.panels.dpi.text);
+		this.calculateSensitivity();
+	}
+	
+	static saveUnitSelection(){
+		$.persistentStorage.setItem('settings.unitSelection', this.panels.unitSelection.GetSelected().id);
+		this.calculateSensitivity();
+	}
+	
+	static loadSettings(){
+		this.panels.dpi.text = $.persistentStorage.getItem('settings.dpi');
+		this.panels.unitSelection.SetSelected($.persistentStorage.getItem('settings.unitSelection'));
+		this.calculateSensitivity();
```

Comment:
These PS keys are a little unclear, I suggest `settings.mouseDPI` and `settings.mouseUnitSelector`